CHANGELOG.md

```diff
@@ -1,3 +1,76 @@
+# 4.0 (alpha-1)
+
+If you're new to the Swift API and migrating from RAC 2, start with the
+[3.0 changes](#30). This section only covers the differences between 3.0 and
+4.0.
+
+ReactiveCocoa 4.0 targets Swift 2 and the current focus is on leveraging the
+improvements from Swift 1.2 to provide a simpler API.
+
+#### Signal operators are protocol extensions
```

Comment:
I think it would be worth mentioning that we brought back `flatten`ing `Signal`s of `SignalProducer`s?